### ISSUE TITLE
Suggestion to add clipboard sync

### ISSUE THREAD

**Issue by cybr47** [localsend/localsend#163]
### Is your feature request related to a problem? Please describe.

App is great. No problem faced yet

### Describe the solution you'd like

Hi, I think clipboard sync or share text between 2 devices would be a great idea to implement in this awesome app.

Thank you

**Comment by Seidko** [comment]
Hello I want to work on this feature. Should I open a new issue to track this feature?

**Comment by ShlomoCode** [comment]
I think the development splits into two tasks:

- Auto-detect and send new copies to paired devices (in OSs where this is possible)
- Auto-copy incoming clips (actually an improvement to the "Auto Finish" option)

**Comment by ManadaHerath** [comment]
Can we create a clipboard in desktop app that holds a similar clipboard to windows or recently mac os, it could be easily accessible if the app supported the clipboard to be available from the localsend menu bar icon in mac os, or in windows system tray from localsend icon. 
It would be 3 simple clicks. 
1. Click the menu icon localsend.
2. Click Clipboard
3. Select the text you wanna copy and click it. (that moves the text from localsend clipboard to os clipboard)

**Comment by ShlomoCode** [comment]
But if there is already a built-in OS clipboard manager, why do we need to implement and maintain our own instead of simply adding the clip to the built-in one?

**Comment by ManadaHerath** [comment]
> But if there is already a built-in OS clipboard manager, why do we need to implement and maintain our own instead of simply adding the clip to the built-in one?

yes its better to add to the clipboard in OS. I suggested a seperate one because I thought the initial implementation will be easier and straightforward because its in localsend application layer and not directly connected with os logics.

**Comment by RWDai** [comment]
Any progress?
